### PR TITLE
Feature#8/crud employee

### PR DESCRIPTION
**Pull request by claunicole**
# Description
-Create the CRUD of the employees
-Create new, edit, and show employee's views
-Create the form for create and edit an employee
-Create the partials for the forms
-Fix the routes of the company's branches and the employees

 # Testing
-Run rails server, go to /employees/new to create a new employee
-In the /employees route you can edit and delete an employee

  ## Screenshots
employees

### CODE REVIEW

File: app/assets/stylesheets/sass/pages/_employees.scss [hunk 1]
```diff
@@ -0,0 +1,36 @@
+.title_empl {
+   text-align: center;
+   margin-top: 200px;
+   color: $white;
+   font-family: "Inter";
+   font-size: 50px;
+   font-style: normal;
+   font-weight: 400;
+   height: 47px;
+   line-height: 47px;
```
Comment (JuanequeX) [inline app/assets/stylesheets/sass/pages/_employees.scss]: Sort alphabetically this block

File: app/assets/stylesheets/sass/pages/_employees.scss [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ .table {
+   border: solid 1px $black;
+   text-align: center;
+   width: 50%;
+ }
+ 
+ .container_table {
+   margin-top: 40px;
+   display: flex;
+   justify-content: center;
```
Comment (JuanequeX) [inline app/assets/stylesheets/sass/pages/_employees.scss]: Sort alphabetically this block

File: app/assets/stylesheets/sass/utils/_forms.scss [hunk 3]
```diff
@@ -0,0 +1,16 @@
+.form-container {
+  padding: 10px;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-content: space-around;
```
Comment (JuanequeX) [inline app/assets/stylesheets/sass/utils/_forms.scss]: Sort alphabetically this block

File: app/views/companies/index.html.erb [hunk 4]
```diff
@@ -3,25 +3,26 @@
...
-      <th scope="row" >Name</th>
-      <th scope="row" >Address</th>
-      <th scope="row" >Id</th> 
-      <th scope="row" ></th> 
-      <th scope="row" ></th>  
+        <th scope="row" >Name</th>
+        <th scope="row" >Address</th>
+        <th scope="row" >Id</th> 
+        <th scope="row" ></th> 
+        <th scope="row" ></th>
```
Comment (JuanequeX) [inline app/views/companies/index.html.erb]: What is the propuse of thi two `<th>` labels?

File: app/views/employees/_form.html.erb [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+        <div class="col-md-8 form-sub">
+          <%= f.label 'private_number' %>
+          <%= f.text_field :private_number, class:"form-title" %>               
+        </div>
+    </div>
+    <div class="form-group"> 
+    <div class="col-md-8 form-sub"> 
+    <%= f.label 'Company' %>
+    <%= collection_select(:employee, :company_id, Company.all, :id, :name, {}, {:multiple => false}) %>
+
```
Comment (JuanequeX) [inline app/views/employees/_form.html.erb]: identend correctly this block

File: app/views/employees/index.html.erb [hunk 6]
```diff
@@ -1,7 +1,28 @@
...
+<h1 class="title title_empl">Empleados</h1>
+<div class="container_table">
+  <table class="table">
+    <thead>
+      <tr>
+        <th scope="row" >Name</th>
+        <th scope="row" >Email</th>
+        <th scope="row" >Id</th> 
+        <th scope="row" ></th> 
+        <th scope="row" ></th>
```
Comment (JuanequeX) [inline app/views/employees/index.html.erb]: What is the propuse of this two labels?

**Review by JuanequeX — CHANGES_REQUESTED** [review]
Please, if they planning to have all your texts in english use english, shouldn't use words in spanish if have labels like: name, edit, delete, etc. in english, make up your mind to use english or spanish.

File: app/assets/stylesheets/sass/pages/_employees.scss (hunk 1 above)
Comment (claunicole) [reply app/assets/stylesheets/sass/pages/_employees.scss]: Done

File: app/assets/stylesheets/sass/utils/_forms.scss (hunk 3 above)
Comment (claunicole) [reply app/assets/stylesheets/sass/utils/_forms.scss]: Done

File: app/views/companies/index.html.erb (hunk 4 above)
Comment (claunicole) [reply app/views/companies/index.html.erb]: It's a space for the edit and delete buttons

File: app/views/employees/_form.html.erb (hunk 5 above)
Comment (claunicole) [reply app/views/employees/_form.html.erb]: Done

File: app/views/employees/index.html.erb [hunk 7]
```diff
@@ -1,7 +1,28 @@
...
+<h1 class="title title_empl">Empleados</h1>
+<div class="container_table">
+  <table class="table">
+    <thead>
+      <tr>
+        <th scope="row" >Nombre</th>
+        <th scope="row" >Email</th>
+        <th scope="row" >Id</th> 
+        <th scope="row" ></th> 
+        <th scope="row" ></th>
```
Comment (YilKanda) [inline app/views/employees/index.html.erb]:
```suggestion
        <th scope="row" >Nombre</th>
        <th scope="row" >Email</th>
        <th scope="row" >Id</th> 
        <th scope="row" colspan="2">Actions</th> 
```

**Review by YilKanda — CHANGES_REQUESTED** [review]

File: app/views/employees/index.html.erb (hunk 7 above)
Comment (claunicole) [reply app/views/employees/index.html.erb]: Thanks, it's done

**Pull request merged**